core/src/test/java/nl/nn/adapterframework/util/XsltStreamingTest.java

```diff
@@ -72,17 +72,17 @@ public void startElement(String uri, String localName, String qName, Attributes
 			print("startElement "+localName);
 			super.startElement(uri, localName, qName, attributes);
 		}
-		
+
 		@Override
 		public void endElement(String uri, String localName, String qName) throws SAXException {
 			print("endElement "+localName);
 			super.endElement(uri, localName, qName);
 		}
 
-		
+
 	}
-	
-	
+
+
 //	@Test()
```

Comment:
Als deze uitgezette test het niet doet, en je ziet dat het al tijden in Git staat, mag je hem ook gewoon verwijderen.